### PR TITLE
Update upgrade-to-different-plan.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the upgrade option as the navigation has changes

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8123

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/commerce/subscriptions/upgrade-to-different-plan.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ When you use the **Upgrade** tab, you're led through the process of buying a new
 
 1. In the Admin center, go to the **Billing** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=842054" target="_blank">Your products</a> page.
 2. Select the subscription that you want to upgrade.
-3. On the subscription details page, select **Upgrade**.
+3. On the subscription details page, Under **Product details and upgrades**, Select **View upgrades recommended for your org**
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/subscriptions/upgrade-to-different-plan.md]:
```suggestion
3. On the subscription details page, under **Product details and upgrades**, select **View upgrades recommended for your org**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder

**Review by cmcatee-MSFT — APPROVED** [review]

**Pull request merged**